src/ProfileEndpoints.jl

```diff
@@ -315,6 +316,26 @@ function handle_task_backtraces(stage_path = nothing)
     return HTTP.Response(200, backtrace_file)
 end
 
+###
+### Profile removal
+###
+
+function has_profile_extension(file)
+    return endswith(file, ".profile") || endswith(file, ".pb.gz") || endswith(file, ".task_backtraces")
```

Comment:
Should the server remember which files were created by it and only delete those? Would multiple server reuse the same stage path?

Reply:
Now that I'm thinking more about it, it might be better to keep this more granular and specify which file we want to remove.

Reply:
Specify it through a query parameter, instead of deleting literally all files.

Reply:
Yes, it's returned as the query result.